echo_test.go

```diff
@@ -428,6 +430,29 @@ func TestEchoStartTLS(t *testing.T) {
 	e.Close()
 }
 
+func TestEchoStartTLSByteString(t *testing.T) {
+	require := require.New(t)
+
+	e := New()
+	e.HideBanner = true
+	defer func() {
+		require.NoError(e.Close())
+	}()
+
+	go func() {
+		cert, err := ioutil.ReadFile("_fixture/certs/cert.pem")
+		require.NoError(err)
+		key, err := ioutil.ReadFile("_fixture/certs/key.pem")
+		require.NoError(err)
+
+		// Prevent the test to fail after closing the servers
+		if err := e.StartTLS(":0", []byte(cert), []byte(key)); err != http.ErrServerClosed {
```

Comment:
https://golang.org/pkg/io/ioutil/#ReadFile

`func ReadFile(filename string) ([]byte, error)`

`[]byte(cert), []byte(key)` is unnecessary conversion